modules/packages/apt_get

```diff
@@ -1,10 +1,23 @@
-#!/usr/bin/python
+#!/bin/sh
+"true" '''\'
+PYTHON_BINS="/usr/bin/python /usr/bin/python3"
+
+for python_bin in $PYTHON_BINS;
+do
+  if [ -f "$python_bin" ]; then
+    exec "$python_bin" "$0" "$@"
```

Comment:
Lol! Yes, that's clever! +1 from me. :-)

But stick to four space indentation, like the rest of the file!